ddl/ddl_test.go

```diff
@@ -75,6 +75,10 @@ func checkEqualTable(c *C, t1, t2 *model.TableInfo) {
 	c.Assert(t1.AutoIncID, DeepEquals, t2.AutoIncID)
 }
 
+func checkHistoryJob(c *C, job *model.Job) {
```

Comment:
Why extract a function to assert ? According the function's name, I cannot understand what will be checked.